src/test/java/io/odh/test/e2e/standard/ModelServingST.java

```diff
@@ -0,0 +1,329 @@
+import io.fabric8.kubernetes.api.model.Namespace;
+import io.fabric8.kubernetes.api.model.NamespaceBuilder;
+import io.fabric8.kubernetes.api.model.Secret;
+import io.fabric8.kubernetes.api.model.SecretBuilder;
+import io.fabric8.kubernetes.client.utils.Serialization;
+import io.fabric8.openshift.api.model.Route;
+import io.fabric8.openshift.api.model.Template;
+import io.fabric8.openshift.client.OpenShiftClient;
+import io.fabric8.openshift.client.dsl.TemplateResource;
+import io.kserve.serving.v1alpha1.ServingRuntime;
+import io.kserve.serving.v1alpha1.ServingRuntimeBuilder;
+import io.kserve.serving.v1alpha1.servingruntimespec.VolumesBuilder;
+import io.kserve.serving.v1alpha1.servingruntimespec.containers.VolumeMountsBuilder;
+import io.kserve.serving.v1alpha1.servingruntimespec.volumes.EmptyDirBuilder;
+import io.kserve.serving.v1beta1.InferenceService;
+import io.kserve.serving.v1beta1.InferenceServiceBuilder;
+import io.odh.test.Environment;
+import io.odh.test.OdhAnnotationsLabels;
+import io.odh.test.OdhConstants;
+import io.odh.test.TestUtils;
+import io.odh.test.framework.listeners.ResourceManagerDeleteHandler;
+import io.odh.test.framework.manager.ResourceManager;
+import io.odh.test.utils.DscUtils;
+import io.odh.test.utils.PodUtils;
+import io.opendatahub.datasciencecluster.v1.DataScienceCluster;
+import io.opendatahub.dscinitialization.v1.DSCInitialization;
+import io.skodjob.annotations.Contact;
+import io.skodjob.annotations.Desc;
+import io.skodjob.annotations.Step;
+import io.skodjob.annotations.SuiteDoc;
+import io.skodjob.annotations.TestDoc;
+import lombok.SneakyThrows;
+import org.hamcrest.Matchers;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.net.ssl.SSLContext;
+import javax.net.ssl.TrustManagerFactory;
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.security.KeyStore;
+import java.security.SecureRandom;
+import java.security.cert.Certificate;
+import java.security.cert.CertificateFactory;
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.Base64;
+import java.util.List;
+import java.util.Locale;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import static io.odh.test.TestConstants.GLOBAL_POLL_INTERVAL_SHORT;
+import static io.odh.test.TestConstants.GLOBAL_TIMEOUT;
+import static io.odh.test.TestUtils.DEFAULT_TIMEOUT_DURATION;
+import static io.odh.test.TestUtils.DEFAULT_TIMEOUT_UNIT;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+@SuppressWarnings({"checkstyle:ClassFanOutComplexity"})
+@SuiteDoc(
+    description = @Desc("Verifies simple setup of ODH by spin-up operator, setup DSCI, and setup DSC."),
```

Comment:
I think that here should be also some mention about model serving, since this describes whole test suite, but I haven't checked result doc file.

Reply:
done